Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/yarrrdrobe.yml

```diff
@@ -0,0 +1,19 @@
+- type: vendingMachineInventory
+  id: YarrrDrobeInventory
+  startingInventory:
+    ClothingNeckCloakPirateCap: 4
+    ClothingNeckCloakPirateParrot: 4
+    ClothingShoesBootsMagPirateFilled: 4
```

Comment:
A few questions here -
1. Does a pirate have access to a bank account?  If so, would it be reasonable to assume they can purchase this?
2. Why would I pay my precious doubloons for magboots (without a knife!) when I could pay spesos for the same thing and use the doubloons for, say, extra cannonballs?